runtime/print.c

```diff
@@ -54,6 +54,10 @@ static void printTagged(FILE *fp, TaggedPtr p, int style, struct PrintStack *sta
         case TAG_INT:
             fprintf(fp, "%" PRId64, taggedToInt(p));
             break;
+        case TAG_FLOAT:
+            // XXX fix precision
+            fprintf(fp, "%g", taggedToFloat(p));
```

Comment:
What's the problem here?

Reply:
If I do `io:println(0.3333333333333333);` it outputs `0.333333`, it defaults to 6 digits. I think I have to figure out how to print minimal amount of digits that will give back the original number when parsed again. Need to look into how to do that.

Reply:
I created #308 for that